libs/messenger-bundle/src/ConnectionEvent/EventFactory.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\MessengerBundle\ConnectionEvent;
+
+use Keboola\MessengerBundle\ConnectionEvent\AuditLog\Organization\ProjectCreatedEvent;
+use Keboola\MessengerBundle\ConnectionEvent\AuditLog\Project\ProjectDeletedEvent;
+use Keboola\MessengerBundle\ConnectionEvent\AuditLog\Project\ProjectFeatureAddedEvent;
+use Keboola\MessengerBundle\ConnectionEvent\AuditLog\Project\ProjectFeatureRemovedEvent;
+use Keboola\MessengerBundle\ConnectionEvent\AuditLog\Project\ProjectUndeletedEvent;
+use Keboola\MessengerBundle\ConnectionEvent\Exception\EventFactoryException;
+use Throwable;
+
+class EventFactory
```

Comment:
Chybi testy

Reply:
Taky jsme doplnil do samostatnyho PR nad verzi vc. GCP https://github.com/keboola/platform-libraries/pull/169. Ta `EventFactory` se prejmenovala na `AuditLogEventFactory`

Reply:
https://github.com/keboola/platform-libraries/pull/169/files#r1365715120

Reply:
Chybejici test jsem pridal. Jestli mu jeste neco chyvi k dokonalosti, rad bych to doresil & doladil v tom #169. Tady v tom PR uz zadna zmena nebude.